apps/api/src/routes/uploads/handlers/create-upload.handler.test.ts

```diff
@@ -0,0 +1,119 @@
+import supertest from "supertest";
+
+import { createServer } from "server";
+import { configs } from "configs";
+
+import { uploadRepository } from "../repositories";
+import * as scraper from "../helpers/scraper";
+
+jest.mock("configs", () => {
+  return {
+    configs: {
+      JWT_SECRET: "just a test",
+      FILE_UPLOAD_MAX_KEYWORD_LIMIT: 4,
+      FILE_UPLOAD_MAX_SIZE: 150,
+    },
+  };
+});
+
+jest.mock("../repositories", () => {
+  return {
+    uploadRepository: {
+      createNew: jest.fn().mockImplementation(() => {}),
+    },
+  };
+});
+
+jest.mock("../helpers/scraper", () => {
+  const original = jest.requireActual("../helpers/scraper");
+
+  return {
+    ...original,
+    scrape: jest.fn().mockImplementation(() => []),
+  };
+});
+
+async function sendUploadKeywordsRequest(path: string): Promise<any> {
+  const app = createServer();
+
+  const token =
+    "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.ZbNYWPJeFZcXiXpyRyNJGJGb2XTblHfHddmTlIFcStQ";
```

Comment:
Will this token even expire?

Reply:
Hi @olivierobert : The token will not expire. So, in this case, the test should always work.

I haven't implemented the token expiration with an `exp` claim in the JWT and token renewing flow this time.